check_brevisone

```diff
@@ -171,6 +171,7 @@ def get_data(base_url, timeout, insecure):
     # Default context for connection
     ctx = ssl.create_default_context()
     if insecure is True:
+        ctx.set_ciphers("DHE-RSA-AES256-GCM-SHA384:@SECLEVEL=1")
```

Comment:
Any reason for this specific cipher? 

If possible, we want to use something as broad as possible, that works for everyone

Reply:
I took this specific cipher due to the output of `openssl s_client --connect smsgw1.invalid.local:443 --cipher 'ALL:@SECLEVEL=1'`

However, i am pretty sure any of the ciphers on the medium level would fit: 
![image](https://github.com/user-attachments/assets/c8a5f114-a52e-4be7-990b-72775d63c38b)

https://docs.brevis.one/current/de/Content/Installation/Setting%20Up%20the%20Gateway/General%20configuration.htm

Reply:
I think that might be too specific for your use case/

```suggestion
        ctx.set_ciphers("ALL:@SECLEVEL=0")
```

If we can do something like this, that would probably work for most users.